filters/ThirdParty/filter_201_WebAnnoyancesUltralist/metadata.json

```diff
@@ -11,5 +11,6 @@
   "tags": [
     "purpose:annoyances"
   ],
-  "trustLevel": "high"
+  "trustLevel": "high",
+  "disabled": true
```

Comment:
Please also change the name like here
https://github.com/AdguardTeam/FiltersRegistry/blob/7638091b16ef2754cabdee61f09735abc5b0ccd2/filters/filter_12_Safari/metadata.json#L3